File: pinyougou-parent/pinyougou-user-interface/src/main/java/com/pinyougou/user/service/UserService.java
package com.pinyougou.user.service;

import com.pinyougou.pojo.TbUser;

public interface UserService {

    void registerUser(TbUser user) throws Exception;

    void createVerifyCode(String phone) throws Exception;

    boolean checkVerifyCode(String phone, String verifyCode) throws Exception;
}

File: pinyougou-parent/pinyougou-user-web/src/main/webapp/plugins/angularjs/service/userService.js
app.service('userService', function ($http) {

    this.register = function (entity, verifyCode) {
        return $http.post('../user/register.do?verifyCode=' + verifyCode, entity);
    };

    this.sendVerifyCode = function (phone) {
        return $http.post('../user/sendVerifyCode.do?phone=' + phone);
    };
});